index.html

```diff
@@ -0,0 +1,349 @@
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="card456">
+      <div class="desk-card-d1">
+        <h2 class="desk-prof-art-dash">Profesional Art Printing Data More</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+
+      <div class="desk-card-d2">
+        <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+
+      </div>
+
+      <div class="desk-card-w1">
+        <h2 class="desk-prof-art-w">Website Protfolio </h2>
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="section-3">
+    <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+        website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+    </div>
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
+      <img class="abt-img-desk" src="./assets/deskimage/Groupme.png" alt="">
```

Comment:
- [ ] Kindly remove this `<img>` tag and use `background-image` property of CSS in `styles.css file` to add the background image. To add a background image it is a best practice to add it using CSS not HTML. You can consider this [reference](https://www.w3schools.com/cssref/pr_background-image.php).
- [ ] ![image](https://user-images.githubusercontent.com/60083437/234991075-0164861b-acfd-49f6-8365-d987f95d1659.png)

- [ ] Kindly add an extra underline below this background image, as per the given design guideline.
- ***Given Design***
- ![image](https://user-images.githubusercontent.com/60083437/234991277-cac57aac-0b6f-4284-ba7c-c5b3e8781996.png)